# Describes the geometry of the turtle's path
#
x: 3.0           # x coord of the lower left corner of the rectangle 
y: 2.0           # y coord of the lower left corner of the rectangle
width: 4.0       # width of the rectangle
height: 5.0      # height of the rectangle
trans_vel: 0.5   # maximum translational velocity of the robot
rot_vel: 0.5     # maximum rotational velocity of the robot
frequency: 60.0 # frequency of the control loop
pentagon_length: 2.0 # Chord length of the pentagon
pentagon_length_real_world: 1.0 # For following waypoint testing

